pkg/patcher/patcher.go

```diff
@@ -46,6 +46,8 @@ func NewSyncerPatcher(ctx *synccontext.SyncContext, pObj, vObj client.Object, op
 		return nil, fmt.Errorf("create virtual patcher: %w", err)
 	}
 
+	vPatcher.direction = "virtual"
+	pPatcher.direction = "host"
```

Comment:
is this going to be required for most patchers that need logging? I'd put it in the constructor if that's the case

Reply:
Sure let me put it in the constructor